From 4b13c42817551d69659b90faf7f004fe29186df4 Mon Sep 17 00:00:00 2001
From: Alan Tsai <alan-tsai@quantatw.com>
Date: Thu, 20 Aug 2026 21:52:38 +0800
Subject: [PATCH 2/2] configurations: revise Terminus 100G NIC configuration.

Add an MCTPI2CTarget entry to allow MCTPReactor to automatically
assign an EID to the BRCM NIC over I2C, removing the need for manual
EID configuration.

Update thresholds

Tested on Rainiera7:
- EID is correctly assigned
- Version object is properly exposed
- Temperature sensor threshold is exposed correct value.
---
 configurations/meta/terminus_100g_nic_tsff.json | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/configurations/meta/terminus_100g_nic_tsff.json b/configurations/meta/terminus_100g_nic_tsff.json
index e4f92d7..a3fefe2 100644
--- a/configurations/meta/terminus_100g_nic_tsff.json
+++ b/configurations/meta/terminus_100g_nic_tsff.json
@@ -14,6 +14,13 @@
             ],
             "Name": "NIC TEMP $bus",
             "Thresholds": [
+                {
+                    "Direction": "less than",
+                    "Index": 2,
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 5
+                },
                 {
                     "Direction": "greater than",
                     "Index": 1,
@@ -30,6 +37,12 @@
                 }
             ],
             "Type": "EMC1403"
+        },
+        {
+            "Address": "0x32",
+            "Bus": "$bus",
+            "Name": "NIC",
+            "Type": "MCTPI2CTarget"
         }
     ],
     "Name": "Terminus NIC FRU $bus",
-- 
2.25.1

